static/app/components/clipboard.tsx

```diff
@@ -43,6 +43,10 @@ function isSupported() {
   return !!document.queryCommandSupported?.('copy');
 }
 
+/**
+ * @deprecated
+ * Replaced by `useCopyToClipboard()` and `<CopyToClipboardButton>`.
+ */
```

Comment:
Do we have any usage in `getsentry`?

Reply:
yes one

Reply:
There are also 3 more instances inside sentry that i couldn't make appear in the ui (so far) so i didn't do the changes.